### DISCUSSION TITLE
THANK YOU ! YOU HAVE NO IDEA HOW HAPPY THIS MADE ME

### DISCUSSION THREAD

**Discussion by ghelanijimmy** [manutalcual/winredock#46] [Show and tell]
Honestly, the simplicity and ease of this tool is amazing! Thank you so much for this! It really helps with that quality of life when using docked/undocked work flows!

**Comment by DanSafdie** [comment]
came here to say the exact same thing. I'm in love! even worked across multiple "desktops." truly gam-echanging. now i can invest in properly arranging my work enviro. Thank you!
